app/controllers/tweet.rb

```diff
@@ -0,0 +1,37 @@
+get '/tweets/index' do
+  # Look in app/views/index.erb
+  @tweets = Tweet.all
+  erb :'/tweets/index'
+end
+
+get '/tweets/new' do
+  @tweets = Tweet.all
+  erb :'/tweets/new'
+end
+
+post '/tweets/new' do
+  puts params
+  # create new tweet by protecting the session
+  @tweet = Tweet.create!(:content => params[:content], :user_id => session[:user_id])
+  if @tweet.save
```

Comment:
Since you're already creating it, the "if save" conditional will not work. Instead of doing Tweet.create! try doing Tweet.new

Reply:
should be fixed on my new branch v2